Casks/m/macgamestore.rb

```diff
@@ -1,20 +1,26 @@
 cask "macgamestore" do
-  version "4.3.4,6084"
-  sha256 "9f4b86ba8726de7c00fa129392b5b3cfdb0af7b682dfedfa1e0b00e08ea0cf11"
+  version "5.3.0,6127"
+  sha256 "3cc7c56c08a78c998ed86ad6b698df2be75045be54bfd3d7db61959ae4291622"
 
-  url "https://www.macgamestore.com/api_clientapp/clientupdates/public/core6/MacGameStore_#{version.csv.first}_#{version.csv.second}.tgz"
+  url "https://www.macgamestore.com/api_clientapp/clientupdates/public/core6/MacGameStore_#{version.csv.first}_#{version.csv.second}.zip"
```

Comment:
```suggestion
  url "https://www.macgamestore.com/api_clientapp/clientupdates/public/core#{version.csv.second.chars.first}/MacGameStore_#{version.csv.first}_#{version.csv.second}.zip"
```

It may just be a coincidence but, based on the versions in this appcast, the number in the `core#` directory name seems to be the first digit of the second part of the version (e.g., `core6` for 4.3.4_6084 and 5.3.0_6127, `core7` for 5.5.0_7014).

Reply:
I can see that being the case. Let's give it a try!